src/components/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,149 @@
+@import "../../styles/global-css-settings";
+@import "../../styles/themes.scss";
+@import "../../styles/typography.scss";
+
+$checkbox-size: 16px;
+
+@mixin hover-selected {
+  background-color: var(--primary-hover-color);
+  border-color: transparent;
+}
+
+@mixin hover-unselected {
+  border-color: var(--color-mud_black);
+}
+
+.monday-style-checkbox {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  height: min-content;
+
+  &:active {
+    .monday-style-checkbox__checkbox {
+      transform: scale(0.8);
+      transform-origin: center;
+      transition:
+              background-color 0ms linear 50ms,
+              border-color 0ms linear 50ms;
+    }
+  }
+
+  &__checkbox {
+    cursor: pointer;
+    visibility: visible;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: $checkbox-size;
+    width: $checkbox-size;
+    border: 1px solid var(--ui-border-color);
+    border-radius: $border-radius-extra-small;
+    transition: transform 50ms cubic-bezier(0, 0, 0.35, 1);
```

Comment:
the transition - we have a sass constat for it

Reply:
something like this - @include transition(transform 50ms cubic-bezier(0, 0, 0.35, 1);  ?